gallery_dl/extractor/luscious.py

```diff
@@ -62,7 +62,10 @@ def get_job_metadata(self):
             ("language", '<p>Language:', ' '),
         ), values={"gallery-id": self.gid})[0]
         data["lang"] = util.language_to_code(data["language"])
-        data["artist"] = text.extract(data["tags"], "rtist: ", ",")[0] or ""
```

Comment:
i can't find the url which caused this error, so i will add another pr if i find that.